onyo/commands/edit.py

```diff
@@ -18,39 +18,56 @@
 logger = logging.getLogger('onyo')
 
 
-def build_commit_cmd(file, git_directory):
-    return ["git -C \"" + git_directory + "\" commit -m", "\'edit " + file + "\'"]
+def build_commit_cmd(files, git_directory):
+    return ["git -C \"" + git_directory + "\" commit -m", "edit files\n" + "\n".join(files)]
 
 
-def edit(args):
-    # set paths
-    git_directory = get_git_root(args.file)
-    full_filepath = get_full_filepath(git_directory, args.file)
-    # path relative from git root dir
-    git_filepath = os.path.relpath(full_filepath, git_directory)
-
-    # check if file is in git
-    run_output = run_cmd("git -C \"" + git_directory + "\" ls-tree -r HEAD ")
-    if git_filepath not in run_output:
-        logger.error(git_filepath + " is not in onyo.")
+def check_sources(sources):
+    problem_str = ""
+    list_of_sources = []
+    if isinstance(sources, str):
+        sources = ["".join(sources)]
+    git_directory = get_git_root(sources[0])
```

Comment:
Registering the outcome of our discussion: commands should not be involved in determining the repo root. This should be done at one point (invocation of onyo) and the result passed down to all commands.